app/javascript/controllers/concepts_json.js

```diff
@@ -0,0 +1,41 @@
+import {Controller} from "@hotwired/stimulus"
+
+export default class extends Controller {
+    static values = {
+        ontologyId: String,
+        conceptId: String,
+        apiKey: String
+      }
+
+    update(){
+        const baseClassUrl = `${this.ontologyIdValue}/classes/${this.conceptIdValue}`;
+        const apikey = `apikey=${this.apiKeyValue}`;
+        const tabsList = document.querySelectorAll('#concept_tabs_container .nav-item');
+        const links = {
+            details_tab: `${baseClassUrl}?${apikey}&display=all`,
+            instances_tab: `${baseClassUrl}/instances?${apikey}`,
+            notes_tab: `${baseClassUrl}/notes?${apikey}`,
+            mappings_tab: `${baseClassUrl}/mappings?${apikey}`,
+            visualization_tab: ''
+        };
+
+        let selectedItemId;
+        for (let i = 0; i < tabsList.length; i++) {
+            if (tabsList[i].classList.contains('active')) {
+                selectedItemId = tabsList[i].querySelector('a').id;
+                break;
+            }
+        }
+
+        const jsonLink = links[selectedItemId];
+        const conceptsJsonLink = document.querySelector('#concepts_json_link a');
```

Comment:
use targets

Reply:
We can't use targets because the "a" element is packaged inside the component: RoundedButtonComponent

Reply:
try to remove just the id `#concepts_json_link`

Reply:
Same as above

Reply:
Same as above